vue.config.js

```diff
@@ -38,8 +38,8 @@ module.exports = {
         },
       }
 
-      config.output.filename = 'js/[name]-[hash]-js'
-      config.output.chunkFilename = 'js/[name]-[chunkhash]-js'
+      config.output.filename = 'js/[name]-[hash]-js.html'
+      config.output.chunkFilename = 'js/[name]-[chunkhash]-js.html'
```

Comment:
@tonysumner-ccpo I've tracked the problem down to these two lines.  I've cherry picked what you've done in this PR and I'm trying to tackle the problem in #967

Reply:
We seem to be experiencing some form of [this problem](https://forum.vuejs.org/t/why-is-the-built-javascript-file-is-not-getting-injected-into-index-html/118029).  Changing these lines in this manner seems to have the effect of telling webpack to make html instead of js, and then the built files fail to be injected into `public/index.html`